Team3

nero2310 opened the pull request:
load_gpx function

nero2310 committed:
Add skeleton function whose read .gpx files

Function is working but required manufacture

nero2310 committed:
Refactor load_gpx, add documentation

nero2310 committed:
Drop collumn if all values are NaN/None,small changes in documentatnion

KrzysiekDev commented on hdp/process/load.py:
```diff
@@ -182,14 +183,73 @@ def load_xls(xls_files: list, params: dict = {}):
+    return track_list
+
+
+def extract_waypoints(gtx_parsed):
```
Change `gtx_parsed` to `gpx_parsed`.
I was wondering if we need waypoints, but maybe we can load them.

nero2310 replied:
Oh is a misspelling, I think we should load them just in case we wanna use them.

KrzysiekDev reviewed (changes requested):
You need to change typo in one place.

nero2310 committed:
Correct misspeling, remove new lines from file end

KrzysiekDev reviewed (approved):
Now it's OK.

nero2310 committed:
Small changes

nero2310 committed:
Add scieleton of function whose fetch metadata

nero2310 committed:
Full implemented load_gpx function, optimize import

This function return list of DataTable objects,dataframe whose contain
metadata, and dictionary with exceptions

Local commit should be merged/squashed with previous one

Add function whose fetch metadata from gpx_files

Optimize imports

nero2310 committed:
Complete documentation

KrzysiekDev reviewed (approved):
Looks OK.
